<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>安装路径:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="installPathlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>721</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="getInstallPathBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取安装路径</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>961</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getDataPathBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取数据路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>数据路径:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataPathlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>721</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="usersComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>721</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>当前用户:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getAllUsersBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取所有用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainBtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始获取内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showMailInfoBtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>显示邮件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showContactsInfoBtn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>70</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>显示联系人</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>961</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>详细内容:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
